shared-bindings/protomatter/Protomatter.c

```diff
@@ -314,10 +314,107 @@ STATIC mp_obj_t protomatter_protomatter_refresh(mp_obj_t self_in) {
 }
 MP_DEFINE_CONST_FUN_OBJ_1(protomatter_protomatter_refresh_obj, protomatter_protomatter_refresh);
 
+//|   .. attribute:: width
+//|
+//|     The width of the display, in pixels
+//|
+STATIC mp_obj_t protomatter_protomatter_get_width(mp_obj_t self_in) {
+    protomatter_protomatter_obj_t *self = (protomatter_protomatter_obj_t*)self_in;
+    check_for_deinit(self);
+    return MP_OBJ_NEW_SMALL_INT(self->width);
```

Comment:
I'd prefer common_hal function calls for these. That means the struct details are contained within common_hal rather than shared-bindings.

Reply:
:+1: